src/test/ui/generic-associated-types/issue-62326-parameter-out-of-range.stderr

```diff
@@ -0,0 +1,15 @@
+error[E0280]: the requirement `for<'a> <Self as Iterator>::Item<'a> : 'a` is not satisfied
+  --> $DIR/issue-62326-parameter-out-of-range.rs:6:1
+   |
+LL |   trait Iterator {
+   |   ^-------------
+   |   |
+   |  _required by `Iterator`
+   | |
+LL | |
+LL | |     type Item<'a>: 'a;
+LL | | }
+   | |_^
```

Comment:
Once this PR lands, can you file a ticket to make this output more visually appealing? The generic handling of spans, not E0280.